llvm/lib/Transforms/Vectorize/LoadStoreVectorizer.cpp

```diff
@@ -119,6 +119,29 @@ using namespace llvm;
 
 #define DEBUG_TYPE "load-store-vectorizer"
 
+cl::opt<bool>
```

Comment:
Avoid adding new cl::opts. If you want these to be configurable they should be pass parameters

Reply:
Got it. We don't really need them, so I'll remove.